packages/app/src/runner/SpecRunnerHeaderRunMode.vue

```diff
@@ -0,0 +1,77 @@
+<template>
+  <div
+    id="spec-runner-header"
+    class="min-h-64px px-16px text-14px"
+    :style="{ width: `${props.width}px` }"
```

Comment:
Seems `props.width` is optional? If so, this needs a conditional, or we might render `width: px`. My suspicion is `width` is  **not** optional.

Reply:
true made required 👍